media/js/src/timer.js

```diff
@@ -42,16 +51,18 @@ const activateTimer = (time) => {
             display_secs = seconds;
         }
         if (minutes === 0 && seconds === 0) {
-            $('#quiz-timer').html('<b>Time is up!</b>');
+            $('#quiz-timer').html('<strong>Time is up!</strong>');
             document.getElementById('quiz_form').submit();
             return;
         }
         $('#quiz-timer')
-            .html(`<b>${display_minutes}:${display_secs}</b>` +
+            .html(`<strong>${display_minutes}:${display_secs}</strong>` +
                  ' remaining to complete quiz.');
 
     }, 1000);
 };
 window.addEventListener('load', function() {
```

Comment:
Just to keep things simple, we don't really need a `load` listener as well as the jQuery ready function I noted above. We should consolidate all the logic into one place - I think the jQuery ready makes sense.

It looks like `load` waits for all stylesheets and images, as well as DOM elements (https://developer.mozilla.org/en-US/docs/Web/API/Window/load_event) - we don't necessarily need to wait for all that here.